### PR TITLE
Add histoneHMM (R package)

### PR DESCRIPTION
**Pull request by dpryan79**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

This adds histoneHMM. This is currently in WIP, since I would prefer to test it a bit beyond what would be done via TravisCI.

### CODE REVIEW

File: recipes/r-histonehmm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  build:
+    - r-base
+    - r-rcpp
+    - bioconductor-genomicranges
+    - bioconductor-rsamtools
+    - r-mvtnorm
+    - r-optparse
+    - bioconductor-biocstyle
+    - posix               # [win]
+    - {{native}}toolchain # [win]
```
Comment (bgruening) [inline recipes/r-histonehmm/meta.yaml]: The toolchain can be removed here in BioConda.

File: recipes/r-histonehmm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    - gcc                 # [not win]
+
+  run:
+    - r-base
```
Comment (bgruening) [inline recipes/r-histonehmm/meta.yaml]: probably add libgcc as well.

File: recipes/r-histonehmm/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    comparison of two samples
+
+# The original metadata for this package was:
+
+#Package: histoneHMM
+#Type: Package
+#Title: Analysis of histone modification data using a HMM
+#Version: 1.6
+#Date: 2013-06-20
+#Author: Matthias Heinig and Maria Colome Tatche
```
Comment (bgruening) [inline recipes/r-histonehmm/meta.yaml]: Remove the comments here.

**Comment by dpryan79** [comment]
Thanks @bgruening!

**Pull request merged**